forest/cli/src/cli/info_cmd.rs

```diff
@@ -0,0 +1,152 @@
+// Copyright 2019-2023 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use std::time::Instant;
+
+use clap::Subcommand;
+use colored::*;
+use forest_blocks::tipset_keys_json::TipsetKeysJson;
+use forest_rpc_client::{
+    chain_get_name, chain_get_tipset, chain_head, start_time, wallet_default_address,
+};
+use forest_shim::econ::TokenAmount;
+use forest_utils::io::parser::{format_balance_string, FormattingMode};
+use time::OffsetDateTime;
+
+use super::Config;
+use crate::cli::handle_rpc_err;
+
+#[derive(Debug, Subcommand)]
+pub enum InfoCommand {
+    Show,
+}
+
+#[derive(Debug, strum_macros::Display)]
+enum SyncStatus {
+    Ok,
+    Slow,
+    Behind,
```

Comment:
When are we `slow` and when are we `behind`?

Reply:
When delta `(current_ts - head.min_timestamp)` is within 5 seconds => `slow`
When delta is > 5 seconds => `behind`.

taken from:
https://github.com/filecoin-project/lotus/blob/bf666a3f7e35317b7400d2059685065eba7ec860/cli/info.go#L215

I've added additional comments at the conditionals.